insight-rest-objects/src/main/java/org/nuxeo/ai/sdk/objects/FieldStatistics.java

```diff
@@ -26,6 +26,18 @@
 
 /**
  * POJO that represents overall statistic for a field
+ *
+ * <code>
+ * {
+ *  "type": "total",
+ *  "field":"total",
+ *  "count":100,
+ *  "total":100,
+ *  "missing":0,
+ *  "cardinality":0,
+ *  "multiClass":false
+ *  }
```

Comment:
We are missing the terms field in this representation.

Reply:
those are just examples. We don't want to cover all scenarios, for that you can look at the documentation.

Reply:
But, this will be THE documentation. where else ?